buildSrc/src/main/kotlin/CodegenTestCommon.kt

```diff
@@ -137,6 +141,7 @@ fun Project.registerGenerateSmithyBuildTask(
     this.tasks.register("generateSmithyBuild") {
         description = "generate smithy-build.json"
         outputs.file(project.projectDir.resolve("smithy-build.json"))
+        // NOTE: This is not working.
```

Comment:
Should we create a TODO + issue for this?

Reply:
filed